rvv-intrinsic-generator/Makefile

```diff
@@ -108,7 +108,7 @@ endef
 define gen_tests
 	rm -f $(1)/*.c
 	$(PY3) -m $(MAIN) --gen $(2) \
-		--out $(1) $(3) \
+		--out $(1) $(3) $(4)\
```

Comment:
No extra parameter is needed in this function. Your patch runs successfully without this edit.